packages/cli/README-source.md

```diff
@@ -301,9 +301,9 @@ zapier convert 1234 --version 1.0.1 my-app
 
 ## Authentication
 
-Most applications require some sort of authentication - and Zapier provides a handful of methods for helping your users authenticate with your application. Zapier will provide some of the core behaviors, but you'll likely need to handle the rest.
+Most applications require some sort of authentication. The Zapier platform provides core behaviors for several authentication methods that might be used with your application, as well as the ability to configure or customize authentication further for certain methods.
 
-> Hint: You can access the data tied to your authentication via the `bundle.authData` property in any method called in your app. Exceptions exist in OAuth and Session auth. Please see them below.
+Data tied to authentication is included in the [bundle object](#bundle-object). Data used on an ongoing basis to authenticate requests is included in `bundle.authData`, while temporary values used in the establishment of [OAuth](#oauth1) and [Session auth](#session) are stored in `bundle.inputData`.
```

Comment:
This still reads a little confusing. During `perform`, all data returned from auth (including when you use oauth1 and session) is in `authData`. 

The only time it's in `inputData` instead is during auth operations. So, for an oauth refresh, the previous auth data is in `inputData`. 

I think the regular `perform` vs `auth operation` (refresh, etc) distinction is important here, so maybe lean on that

Reply:
Thanks for the analysis on this! The point about the existing authData going into inputData for OAuth refresh does mean that my framing isn't quite there. I guess I think of `perform` as also being an "auth operation", so I'm still not sure if that will improve clarity overall. 🤔 I'll give this some more thought.